src/test/java/unicash/model/ModelManagerTest.java

```diff
@@ -131,7 +131,49 @@ public void getExpenseSummary_success() {
         expectedExpenseSummary.put("ta", 888.8);
         expectedExpenseSummary.put("food", 8.8);
 
-        assertEquals(modelManager.getExpenseSummary(), expectedExpenseSummary);
+        assertEquals(modelManager.getExpenseSummaryPerCategory(), expectedExpenseSummary);
+    }
+
+    @Test
+    public void hasExpenses_noExpenses_success() {
```

Comment:
Could the `success` be replaced with the specific outcome? Since there's different potential outcomes so might be good to include something like `returnsFalse`